tests/pipelines/kit-cluster/pipeline-template.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
file name stutters with folder name ? Why are we doing this ?
Also, what's the point of this pipeline ? It just sets-up KIT CP, DP and tears down, am i missing something here ?

Reply:
This is a template and we will write a tutorial about how to extend it to run tests. I agree that the folder name stutters. I will rename it as template.yaml.

Reply:
can we have a more meaningful example where we do something like run a test or scale upto `n` pods when we bring up kit cluster that Cx can refer to instead of this ?

Reply:
This template is used as the minimal example to use the modules of the Tekton pipeline. We discussed about what type of getting start example we should provide. There are two items:

-  **Minimal setup with minimal parameters:** This is to demonstrate that the simplest test requires very little hand holding. 
 
-  **Modular design:** We need to demonstrate how the Tekton module works. So instead of us providing a payload like scaling upto `n` pods,  what we will do is to create a new task in the tutorial from scratch and teach them how to insert it into this pipeline template.

Therefore, we will try to keep the pipeline as bare-bone as possible